README.md

```diff
@@ -344,146 +344,124 @@ to get started setting up your self-hosted runner.
 ### Allocating cloud resources with CML
 
 When a workflow requires computational resources (such as GPUs) CML can
-automatically allocate cloud instances. For example, the following workflow
+automatically allocate cloud instances using `cml-runner`. You can spin up instances on your AWS or Azure account (GCP support is forthcoming!). 
+
+For example, the following workflow
 deploys a `t2.micro` instance on AWS EC2 and trains a model on the instance.
-After the instance is idle for 120 seconds, it automatically shuts down.
+After the job runs, the instance automatically shuts down.
 
 ```yaml
-name: train-my-model
+name: "Train-in-the-cloud"
 on: [push]
+
 jobs:
-  deploy-cloud-runner:
+  deploy-runner:
     runs-on: [ubuntu-latest]
-    container: docker://dvcorg/cml:latest
     steps:
+      - uses: hashicorp/setup-terraform@v1
+      - uses: iterative/setup-cml@v1
+      - uses: actions/checkout@v2
       - name: deploy
+        shell: bash
         env:
-          repo_token: ${{ secrets.REPO_TOKEN }}
+          repo_token: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
         run: |
-          echo "Deploying..."
-
-          MACHINE="cml$(date +%s)"
-          docker-machine create \
-              --driver amazonec2 \
-              --amazonec2-instance-type t2.micro \
-              --amazonec2-region us-east-1 \
-              --amazonec2-zone f \
-              --amazonec2-vpc-id vpc-06bc773d85a0a04f7 \
-              --amazonec2-ssh-user ubuntu \
-              $MACHINE
-
-          eval "$(docker-machine env --shell sh $MACHINE)"
-
-          (
-          docker-machine ssh $MACHINE "sudo mkdir -p /docker_machine && sudo chmod 777 /docker_machine" && \
-          docker-machine scp -r -q ~/.docker/machine/ $MACHINE:/docker_machine && \
-
-          docker run --name runner -d \
-            -v /docker_machine/machine:/root/.docker/machine \
-            -e RUNNER_IDLE_TIMEOUT=120 \
-            -e DOCKER_MACHINE=${MACHINE} \
-            -e RUNNER_LABELS=cml \
-            -e repo_token=$repo_token \
-            -e RUNNER_REPO="https://github.com/${GITHUB_REPOSITORY}" \
-           dvcorg/cml-py3 && \
-
-          sleep 20 && echo "Deployed $MACHINE"
-          ) || (echo y | docker-machine rm $MACHINE && exit 1)
-  train:
-    needs: deploy-cloud-runner
-    runs-on: [self-hosted, cml]
-
+          cml-runner \
+          --cloud aws \
+          --cloud-region us-west \
+          --cloud-type=t2.micro \
+          --labels=cml-runner
+  train-model:
```

Comment:
A few thoughts to make it easier to see the connection between this example and the ones above:
1. Does the job order matter? If not, can this job be first to put the familiar parts before the new ones?
2. Can either this job or the `run` jobs above be renamed so they all have the same name?
3. There are a few part where the job details don't match the above examples, and it's unclear if there's a reason for the difference. I'll try to point out below.

Reply:
Unfortunately, job order matters a lot- the job `train-model` requires `deploy-runner` to have been run first. If we switch it up the code won't work. But maybe I can add a sentence that gently reminds folks this is largely identical to what we've seen in the basic examples, with the addition of the `cml-runner` function as its own job.

We can be more consistent with the names throughout the readme- thanks for pointing that out.